todo.md

```diff
@@ -1,34 +1,19 @@
 ## TODO:
 
 1. search for "// TODO:#checkout" (leftovers that can't be done until cart is merged)
-2. catch refresh bug on b2b (account type on refresh displays the payment method step when it's not supposed to) - related to b2b guard issue (partially solved until further discussions)
-3. BUG - When doing b2b checkout: - related to multiple cms trigger issue (partially solved until further discussions)
-   1. select the credit card payment method
-   2. go all the way to the last step (review order)
-   3. notice the delivery method is there
-   4. go back to the previous step (payment), and refresh browser
-   5. go to the last step - the delivery mode is not there.
-4. BUG - the payment method infinite spinner: - related to multiple cms trigger issue (partially solved until further discussions)
-   1. repeat the steps above
-   2. go to the first step (payment method)
-   3. notice the infinite spinner
-5. In b2b, add a spinner to the 1st checkout step (payment method). The reason is the bug: 
-   1. Have the account type selected
-   2. Switch to credit card
-   3. if we're quick enough, and click "continue" button before the two API calls are resolved, we'll see _4_ steps instead of _5_.
-6. Is the checkout properly using the new cart lib?
-   1. order and repl order confirmation page context: https://github.com/SAP/spartacus/pull/14466/files (source: https://sap-cx.slack.com/archives/C02L8BUATM5/p1638282843004200) - related to waiting for Wei and Patrick PR to be merged (partially solved until further discussions)
+2. Is the checkout properly using the new cart lib? (still waiting for cart-lib PR to be merged)
+   1. order and repl order confirmation page context: https://github.com/SAP/spartacus/pull/14466/files (source: https://sap-cx.slack.com/archives/C02L8BUATM5/p1638282843004200) - related to waiting for Wei and Patrick PR to be merged 
    - (related to above) sample data changes: https://github.tools.sap/cx-commerce/spartacussampledata/pull/211 (source: https://sap-cx.slack.com/archives/C02L8BUATM5/p1638283007005900)
-7.  check the event listeners for the following scenario:
+3.  check the event listeners for the following scenario:
     1.  a user started the checkout, entered their delivery address, and set the delivery mode, and the data is sent on the back-end for the active cart
     2.  the user changes their mind, and navigates away from the checkout page to homepage, and refreshes the browser.
     3.  after it, they decide to change their address in the profile menu. 
     4.  if they now start the checkout (and LL the feature), the current back-end data is _not_ valid for the active cart - we must reset the set delivery mode, and load the supported delivery modes again for the new address.
     5.  if the listener was in the root module, it can listen to the userupdateaddress event, ll the checkout, and issue a reset query event
```

Comment:
I just checked this, and it seems to be working as expected, so we can remove it

Reply:
Yep, thanks!